storage/storage.go

```diff
@@ -113,13 +113,47 @@ var (
 			return 1
 		`)
 
+	// Atomically get an aggregation
+	//
+	// This should only return an aggregation if it is available for processing.
+	// This is determined by the ExpiresAt key which contains a Unix Timestamp.
+	// ExpiresAt:
+	//   - false                        : Aggregation not found
```

Comment:
In which cases an aggregation may not be found and the **ExpiresAt** field be set to False? Could you provide some examples please?

Reply:
This is returned when `redis.call()` fails to execute. It is unlikely but we should handle all cases imho

Reply:
Yes, totally agree with handling all cases, I just couldn't think of a case where false will be returned, thanks!